Update WoT Tutorial Website: Building Blocks Page (#207)
w3c/wot-cg: Tutorials/whatiswot/website/docs/wot/building-blocks.md

<br />
<br />

With this tutorial, we begin to explain the concepts specific to the Web of Things. Be sure to review the [Preliminary tutorial](/docs/preliminary/payloads) to understand what the Web of Things builds upon. Here, we’ll provide a brief overview of the main building blocks that make up the Web of Things architecture — Thing Description, Binding Templates, Discovery, Scripting API, and Profiles. In separate later tutorials, we will explore each individual building block in more detail.

## What are building blocks?

Building blocks allow the implementation of systems that conform with the abstract WoT Architecture. Let us take a closer look at the WoT building blocks. 

> ![wot-building-blocks](/img/12-Building-Blocks/wot-building-blocks-new.png)

Reviewer: the previous one can be deleted and this one used

<br />
<br />

> This is the first tutorial in our Web of Things series. Here, we’ll provide a brief overview of the main building blocks that make up the Web of Things architecture — Thing Description, Binding Templates, Discovery, Scripting API and Profiles. In separate later tutorials, we will explore each individual building block in more detail.

Reviewer: ```suggestion
With this tutorial, we begin to explain the concepts specific to the Web of Things. Be sure to review the Preliminary tutorials (ADD LINK) to understand what the Web of Things builds upon. Here, we’ll provide a brief overview of the main building blocks that make up the Web of Things architecture — Thing Description, Binding Templates, Discovery, Scripting API and Profiles. In separate later tutorials, we will explore each individual building block in more detail.
```

## What are building blocks?

Building blocks allow the implementation of systems that conform with the abstract WoT Architecture. Let us take a closer look at the WoT building blocks.

![wot-building-blocks](/img/12-Building-Blocks/wot-building-blocks-new.png)

> A Thing always has a Thing Description. It is a key building block that provides metadata describing a Thing and its network interface. This TD can be shared directly by the Thing or through the TD Directory — which is what we call Discovery. Things and Consumers can be programmed with the Scripting API.

Reviewer: ```suggestion
A Thing always has a Thing Description (TD). It is a key building block that provides metadata describing a Thing and its network interfaces. This TD can be hosted directly by the Thing or through a TD Directory — which is what we call Discovery. Things and Consumers can be programmed with the Scripting API.
```

## Thing Description

Let us begin with the Thing Description, or TD — a key component of WoT.

All TDs can be processed just like normal JSON documents. Essentially, a TD defines an information model using a semantic vocabulary and a serialized JSON representation. It provides both human- and machine-readable metadata describing the capabilities of a Thing.

> A TD describes a Thing instance with general metadata such as name, ID, and description. It typically also contains Interaction Affordance metadata and Protocol Bindings (such as Modbus and HTTP), as well as both public and local IP addresses.

Reviewer: ```suggestion
A TD describes a Thing instance with general metadata such as name, ID, and description. It typically also contains Interaction Affordance metadata and Protocol Bindings, such as Modbus or HTTP.
```

IP address point is correct but difficult to explain without explaining it in detail
